networklibrary/src/main/java/com/eis/communication/network/INetworkDictionary.java

```diff
@@ -0,0 +1,32 @@
+package com.eis.communication.network;
+
+/**
+ * Interface to define a network dictionary
+ * @param <K> The Key for each resource
+ * @param <R> The Resource
+ *
+ * @author Marco Cognolato
+ */
+public interface INetworkDictionary<K,R>{
```

Comment:
Ew, `I` before interfaces name, how C#-ish.

Reply:
In fact I come from a mainyl C# language. Anyway, I think using the `I` would be better because you can instantly distinguish an interface from a class.
I mean, would you easily understand that `NetworkManager` was an interface?

Reply:
https://stackoverflow.com/questions/2814805/java-interfaces-implementation-naming-convention
> Putting I in front is just Hungarian style notation tautology that adds nothing but more stuff to type to your code

https://howtodoinjava.com/java/basics/java-naming-conventions/

I'm not a fan of "I" at all. Also only these two interfaces changed name? Either you change them all (shivers down my back) or you call this propely by removing the I.

Reply:
> adds nothing but more stuff to type to your code

1 letter, lol
Also no, gives more info on what you're using. Anyway, I'll remove the `I`s